charts/scalardb-cluster/values.yaml

```diff
@@ -317,3 +317,17 @@ scalardbCluster:
         - localhost
       # -- Issuer references of cert-manager.
       issuerRef: {}
+
+  encryption:
+    # -- Enable encryption feature. You must enable encryption when you use the encryption feature of ScalarDB Cluster.
+    enabled: false
+    # -- Type of encryption. You must set the same value as "scalar.db.cluster.encryption.type" of ScalarDB Cluster.
+    type: ""
+    # -- Vault specific configurations.
+    vault:
+      # -- TLS configurations to access from ScalarDB Cluster to Vault by using TLS.
+      tls:
+        # -- Enable TLS between ScalarDB Cluster and Vault. Note that you must enable the TLS feature on the Vault side.
+        enabled: false
+        # -- Name of the Secret containing the custom CA root certificate for TLS communication between ScalarDB Cluster and Vault. The certificate file will be mounted under the /encryption/vault/tls/certs/ directory in the ScalarDB Cluster pod.
```

Comment:
Please update the comments in this section based on the changes to the tables with the same wording in `charts/scalardb-cluster/README.md`. I didn't make suggestions for each one here, just in case we need to change the wording that I suggested.